source/common/quic/envoy_quic_server_session.cc

```diff
@@ -37,6 +38,13 @@ absl::string_view EnvoyQuicServerSession::requestedServerName() const {
   return {GetCryptoStream()->crypto_negotiated_params().sni};
 }
 
+void EnvoyQuicServerSession::dumpState(std::ostream& os, int indent_level) const {
+  const char* spaces = spacesForLevel(indent_level);
+  os << spaces << "Http3::ServerSession";
```

Comment:
Can we dump the pointer to "this" as done for the other implementations of this e,g: https://github.com/envoyproxy/envoy/blob/ff24017783df8cb0f342d7532b565bd0cd54f106/source/common/http/http2/codec_impl.cc#L1893 


Seems like you might  want to dump the QUIC connection? Will defer to @RyanTheOptimist  for what else should be dumped here. https://github.com/envoyproxy/envoy/blob/2757eb9eb0faece7b2779f7c1e21953b31b836f4/source/common/quic/envoy_quic_server_session.h#L117